docs/topics/tour/kotlin-tour-basic-types.md

```diff
@@ -0,0 +1,96 @@
+[//]: # (title: Basic types)
+
+<microformat>
+    <p>This is the second part of the <strong>Beginner</strong> Kotlin tour:</p>
+    <p><img src="icon-1-done.svg" width="20" alt="First step" /> <a href="kotlin-tour-hello-world.md">Hello world</a><br />
+        <img src="icon-2.svg" width="20" alt="Second step" /> <strong>Basic types</strong><br />
+        <img src="icon-3-todo.svg" width="20" alt="Third step" /> <a href="kotlin-tour-collections.md">Collections</a><br />
+        <img src="icon-4-todo.svg" width="20" alt="Fourth step" /> <a href="kotlin-tour-control-flow.md">Control flow</a><br />
+        <img src="icon-5-todo.svg" width="20" alt="Fifth step" /> <a href="kotlin-tour-functions.md">Functions</a><br />
+        <img src="icon-6-todo.svg" width="20" alt="Sixth step" /> <a href="kotlin-tour-classes-part-1.md">Classes</a><br />
+        <img src="icon-7-todo.svg" width="20" alt="Final step" /> <a href="kotlin-tour-null-safety.md">Null safety</a></p>
+</microformat>
+
+Kotlin has data types that tell the compiler what functions and properties an object has.
+
+In the last chapter, Kotlin's powerful type inference inferred from the previous example that `customers` has type: [`Int`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/).
+As `Int` is a number data type, you can perform arithmetic operations with `customers`:
```

Comment:
I feel the need to start with the traditional arithmetic operations like `+`/`-` and then go with augmented assignments.